consensus/vbft/event_timer.go

```diff
@@ -190,7 +187,7 @@ func (self *EventTimer) getEventTimeout(evtType TimerEventType) time.Duration {
 //
 // internal helper, should call with lock held
 //
-func (self *EventTimer) startEventTimer(evtType TimerEventType, blockNum uint32) error {
+func (self *EventTimer) startEventTimer(evtType TimerEventType, blockNum uint32) {
```

Comment:
suggest changing the panic to 'return err'

Reply:
ok